modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="lvms-unstalling-lvms-with-web-console_{context}"]
-= Uninstalling {lvms} installed using the OpenShift Web Console
+= Uninstalling {lvms} installed using the web console
 
-You can unstall {lvms} using the Red Hat OpenShift Container Platform Web Console.
+You can uninstall {lvms} using the OpenShift Container Platform web console.
```

Comment:
```suggestion
You can uninstall {lvms} using the {product-title} web console.
```
Missed this on my first time through @sr1kar99

Reply:
Updated as per the suggestion.